serverless.yml

```diff
@@ -91,7 +92,8 @@ provider:
 
     - Effect: Allow
       Action:
-        - redshift:ModifyCluster
+        - sns:ListTopics
+        - sns:Publish
```

Comment:
Also, is there a reason for  sns:Publish & sns:ListTopics permission ?

Reply:
No.  A mistake in editing.